tests/cupyx_tests/scipy_tests/interpolate_tests/test_polyint.py

```diff
@@ -0,0 +1,210 @@
+
+    @testing.for_all_dtypes(no_bool=True, no_float16=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-5, rtol=1e-5)
+    def test_scalar(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = numpy.poly1d([-1, 2, 6, -3, 2])
+        xs = numpy.linspace(-1, 1, 10, dtype=dtype)
+        ys = true_poly(xs)
+        if xp is cupy:
+            xs = cupy.asarray(xs)
+            ys = cupy.asarray(ys)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        return P(xp.array(7, dtype=dtype))
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_derivatives(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        D = P.derivatives(test_xs)
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_low_derivatives(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        D = P.derivatives(test_xs, len(xs) + 2)
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_derivative(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        m = 10
+        D = P.derivative(test_xs, m)
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_high_derivative(self, xp, scp, dtype):
+        if xp.dtype(dtype).kind in 'iu':
+            pytest.skip()
+        true_poly = xp.poly1d([-2, 3, 1, 5, -4])
+        test_xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        xs = xp.linspace(-5, 5, 5, dtype=dtype)
+        ys = true_poly(xs)
+        P = scp.interpolate.KroghInterpolator(xs, ys)
+        D = P.derivative(test_xs, 2 * len(xs))
+        return D
+
+    @testing.for_all_dtypes(no_bool=True, no_complex=True)
+    @testing.numpy_cupy_allclose(scipy_name='scp', atol=1e-7, rtol=1e-7)
+    def test_hermite(self, xp, scp, dtype):
```

Comment:
Although I know SciPy tests the same in `test_hermite` and `test_lagrange`, I want to have a test of Hermite interpolation here, that is, want to give some derivative value as well as observed value to construct the interpolator.

SciPy's KroghInterpolator.derivatives() has a clear example to give derivatives to construct an interpolator, where observed(0th), 1th, and 2nd values at x = 0 are given.
https://docs.scipy.org/doc/scipy/reference/generated/scipy.interpolate.KroghInterpolator.derivatives.html#scipy.interpolate.KroghInterpolator.derivatives

Reply:
Interesting! Added these cases. Thanks for letting me know :)